prisma/new/schema.prisma

```diff
@@ -0,0 +1,823 @@
+  syllabusRateTranslations     SyllabusRateTranslation[]
+  syllabusProgressTranslations SyllabusProgressTranslation[]
+  syllabusOfficeTimes          SyllabusOfficeTime[]
+}
+
+/// 可以找到老師或助教的時間
+// ceiba aca_office_time
+model SyllabusOfficeTime {
+  syllabusId Int
+  syllabus   Syllabus @relation(fields: [syllabusId], references: [id])
+  /// 星期幾
+  week       Int
+  /// 開始時間
+  // ex: 09:15
+  startAt      String   @db.VarChar(5)
+  // 結束時間
+  endAt        String   @db.VarChar(5)
+  createdAt  DateTime @default(now())
+
+  @@unique([syllabusId, week, startAt, endAt])
+}
+
+model SyllabusTranslation {
+  language          Language
+  syllabusId        Int
+  syllabus          Syllabus @relation(fields: [syllabusId], references: [id])
+  // aca_course_goal table
+  /// 課程概述
+  describe          String?
+  /// 課程目標
+  goal              String?
+  /// 課程要求
+  // 原background
+  requirement       String?
+  // 預計課後學習時數
+  workLoad          String?
+  // 指定閱讀
+  designatedReading String?
+  // aca_course_book
+  /// 參考書目
+  book              String?
+
+  createdAt DateTime @default(now())
+
+  @@id([language, syllabusId])
+}
+
+/// 大綱 評分方式
+// ceiba.aca_course_req
+model SyllabusRateTranslation {
+  language   Language
+  syllabusId Int
+  syllabus   Syllabus @relation(fields: [syllabusId], references: [id])
+  /// 評分項目
+  content    String   @db.VarChar(60)
+  percent    Int
+  note       String?
+
+  createdAt DateTime @default(now())
+
+  // ex: 上課參與 / 60%  / 出席、導讀、報告與討論 
+  @@id([language, syllabusId])
+}
+
+/// 大綱 課程進度
+// ceiba.aca_course_syllabus
+model SyllabusProgressTranslation {
+  language   Language
+  syllabusId Int
+  syllabus   Syllabus @relation(fields: [syllabusId], references: [id])
+  week       Int
+  // ex: string 11/8
+  date       String
+  // 原subject
+  content    String
+  // 過了設定的期限才會顯示給同學看見
+  publicDate DateTime
+
+  createdAt DateTime @default(now())
+
+  @@id([language, syllabusId])
+}
+
+// ########################################
+// User
+// ########################################
+model User {
+  id                       String                     @id @default(uuid()) @db.Uuid
+  // 人事室教職員編號 5碼 或 學生學號 9碼
+  sequence                 String
+  userType                 UserType
+  departmentId             String?                    @db.VarChar(4)
+  department               Department?                @relation(fields: [departmentId], references: [id])
+  /// 年級
+  grade                    Int?
+  createdAt                DateTime                   @default(now())
+  updatedAt                DateTime                   @updatedAt
+  courseCollections         CourseCollection[]
+  userInfoTranslations      UserInfoTranslation[]
+  preseletectedCourseTables PreseletectedCourseTable[]
+}
+
+model UserInfoTranslation {
+  language  Language
+  userId    String   @db.Uuid
+  user      User     @relation(fields: [userId], references: [id])
+  name      String
+  /// 組別， 只有學生才有
+  // ex: 國際關係組
+  group     String?
```

Comment:
As I remember, the department name is not separating the department name and group name, so I might be - `{ "name": "政治學系國際關係組" }` instead of `{ "name": "政治學系", "group": "國際關係組" }`. Please make sure the data payload before we create it.

Reply:
這個部份我會再去確認
這邊需要申請學生資料的DB訪問權限
可能沒這麼快

Reply:
剛剛有去確認學生學籍資料庫
我這邊的name是學生名稱
有的學生填基本資料時有填英文名字
或是外籍生有取中文名字

group的話會是 政治系國際關係組 or DEPARTMENT OF POLITICAL SCIENCE, INTERNATIONAL RELATIONS DIVISION                                 的i18n